app/gui/src/controller/searcher/breadcrumbs.rs

```diff
@@ -60,12 +63,13 @@ impl Breadcrumbs {
     /// Returns a currently selected breadcrumb id. Returns [`None`] if the top level breadcrumb
     /// is selected.
     pub fn selected(&self) -> Option<suggestion_database::entry::Id> {
-        if self.is_top_module() {
-            None
-        } else {
-            let index = self.selected.get();
-            self.list.borrow().get(index - 1).map(BreadcrumbEntry::id)
-        }
+        let index = self.selected.get();
+        self.list.borrow().get(index).map(BreadcrumbEntry::id)
+    }
+
+    /// Return a list of breadcrumbs to be displayed in the panel.
+    pub fn items(&self) -> impl Iterator<Item = BreadcrumbEntry> + '_ {
```

Comment:
It looks suspicious lifetime-wise, and it seems to be unused.

Reply:
Removed.